src/layers/schematicInterfaces.ts

```diff
@@ -105,6 +106,124 @@ export interface Cement {
   toc: number;
 }
 
+export type PerforationSubKind =
+  | 'Perforation'
+  | 'Open hole gravel pack'
+  | 'Open hole screen'
+  | 'Open hole'
+  | 'Open hole frac pack'
+  | 'Cased hole frac pack'
+  | 'Cased hole gravel pack'
+  | 'Cased hole fracturation';
+
+export interface Perforation {
+  kind: 'perforation';
+  subKind: PerforationSubKind;
+  id: string;
+  top: number;
+  bottom: number;
+  isOpen: boolean;
```

Comment:
Maybe we should add a typedoc to `isOpen` so the users knows what it's referencing.

Reply:
I don't know how to do this. Could you talk to me in the office about this one today @ovineq ?